cogs/push.py

```diff
@@ -52,13 +53,17 @@ def push_data(cogs_dir, spreadsheet, tracked_sheets, remote_sheets):
         if not os.path.exists(sheet_path):
             logging.warning(f"'{sheet_title}' exists remotely but has not been pulled")
             continue
-        with open(sheet_path, "r") as f:
-            reader = csv.reader(f, delimiter=delimiter)
-            for row in reader:
-                row_len = len(row)
-                if row_len > cols:
-                    cols = row_len
-                rows.append(row)
+        with open(sheet_path, "r") as fr:
+            reader = csv.reader(fr, delimiter=delimiter)
+            tracked_sheet_title = re.sub(r"[^A-Za-z0-9]+", "_", sheet_title.lower())
```

Comment:
If this regex and the path on the next line are often reused, I'd rather have shared helper functions for them.

Reply:
Done